### PR TITLE
Orlp 240 fix Sonar issues Part-4 Controllers

### PR DESCRIPTION
**Pull request by Tamara20**

### CODE REVIEW

File: src/main/java/com/softserve/academy/spaced/repetition/dto/impl/PasswordDTO.java [hunk 1]
```diff
@@ -26,8 +26,10 @@ public boolean equals(Object o) {
         if (!(o instanceof PasswordDTO)) return false;
```
Comment (antonbakalets) [inline src/main/java/com/softserve/academy/spaced/repetition/dto/impl/PasswordDTO.java]: Sonar should report that curly brackets are missing.

Comment (Tamara20) [reply src/main/java/com/softserve/academy/spaced/repetition/dto/impl/PasswordDTO.java]: Sonar want: Replace this if-then-else statement by a single return statement. Is it need to do in one if statement?

Comment (antonbakalets) [reply src/main/java/com/softserve/academy/spaced/repetition/dto/impl/PasswordDTO.java]: You are talking about line 29. https://sbforge.org/sonar/rules/show/squid:S1126?layout=false

Comment (antonbakalets) [reply src/main/java/com/softserve/academy/spaced/repetition/dto/impl/PasswordDTO.java]: Looks like curly brackets are not forced by Sonar, it is a rule on my current project.

File: src/main/java/com/softserve/academy/spaced/repetition/dto/impl/UserForPersonalPage.java [hunk 2]
```diff
@@ -0,0 +1,4 @@
+package com.softserve.academy.spaced.repetition.dto.impl;
+
+public class UserForPersonalPage {
```
Comment (SolukAndrian) [inline src/main/java/com/softserve/academy/spaced/repetition/dto/impl/UserForPersonalPage.java]: Why you add this empty class?

File: src/main/java/com/softserve/academy/spaced/repetition/dto/impl/PasswordDTO.java (hunk 1 above)
Comment (Tamara20) [reply src/main/java/com/softserve/academy/spaced/repetition/dto/impl/PasswordDTO.java]: Yes I about it, so I fix curly brackets

**Pull request merged**